src/agent/protocols/protos/image.proto

```diff
@@ -22,6 +22,8 @@ message PullImageRequest {
     string container_id = 2;
     // Use USERNAME[:PASSWORD] for accessing the registry
     string source_creds = 3;
+    // Use skopeo or rust crate to download container image
+    bool use_skopeo = 4;
```

Comment:
I wonder if we should invert this and make it something like:

```suggestion
    // If set use an external helper application (such as skopeo) to download container image, otherwise use the built-in capability.
    bool use_external_download_helper = 4;
```

Reply:
ok, I'll use with this way.